Makefile.am

```diff
@@ -214,35 +214,35 @@ dist_doc_DATA = \
     AUTHORS \
     LICENSE \
     README.md \
-    doc/README.DGUX386 \
-    doc/README.Mac \
-    doc/README.OS2 \
-    doc/README.aix \
-    doc/README.amiga \
-    doc/README.arm.cross \
+    doc/platforms/README.DGUX386 \
```

Comment:
Should we also update cmake script?

Reply:
It looks as if it uses a glob so it seemed to still work for me...